chia/wallet/wallet_nft_store.py

```diff
@@ -157,6 +157,19 @@ async def count(self, wallet_id: Optional[uint32] = None, did_id: Optional[bytes
                 return int(count_row[0])
         return -1
 
+    async def is_empty(self, wallet_id: Optional[uint32] = None) -> bool:
+        sql = "SELECT 1 FROM users_nfts WHERE removed_height is NULL"
```

Comment:
does `1` refer to column 1?
would it make sense to query `COUNT(*)` instead? (still with `LIMIT 1`). It might be optimized and would require less data to be passed back from sqlite and the aiosqlite thread

Reply:
it returns `1` as a constant for each row that is returned, so it's more performant than `COUNT`